### PR TITLE
add pr template

### PR DESCRIPTION
**Pull request by rwmcintosh**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+## Type of change
+
+Please mark relevant options with an `x` in the brackets.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
```
[the remark this answers is not in the document]
Comment (rwmcintosh) [reply .github/PULL_REQUEST_TEMPLATE.md:14]: Why don't we make it like 

- [ ] This change requires documentation changes (link at least one documentation issue):

and not put checkboxes for the list.

[the remark this answers is not in the document]
Comment (Yuri05) [reply .github/PULL_REQUEST_TEMPLATE.md:14]: fine with this

**Comment by Yuri05** [comment]
@rwmcintosh Please also add **PULL_REQUEST_TEMPLATE.md** to the exclusions list of appveyor.yml

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rwmcintosh) [reply .github/PULL_REQUEST_TEMPLATE.md:14]: oops, I hadn't made that change yet

**Comment by rwmcintosh** [comment]
Isn't that what this does:
```
skip_commits:
  files:
    - "*.md"
```

**Comment by Yuri05** [comment]
> Isn't that what this does:
> 
> ```
> skip_commits:
>   files:
>     - "*.md"
> ```

I think it ignores .md on the **top level only**, while **/*.md would exclude all .md recursively.

**Pull request merged**